python/samples/getting_started/agents/ollama/ollama_with_openai_chat_client.py

```diff
@@ -52,6 +53,7 @@ async def streaming_example() -> None:
     print("=== Streaming Response Example ===")
 
     agent = OpenAIChatClient(
+        api_key=os.getenv("OLLAMA_API_KEY"),
```

Comment:
You can change this too like you did for the non-streaming example
```suggestion
        api_key="ollama",
```